migration/sql-files/041-unique-area-name-create-new-area.sql

```diff
@@ -0,0 +1,80 @@
+------ You can't allow the same area name and the same ancestry inside a space
+
+ALTER TABLE areas ADD CONSTRAINT areas_name_space_id_path_unique UNIQUE(space_id,name,path);
+
+------  For existing spaces in production, which dont have a default area, create one.
```

Comment:
Please move comment for a function inside the function so we can see it when inspecting the function from psql directly.